utils.py

```diff
@@ -48,6 +53,20 @@
 SECONDS_IN_HOUR = 60 * 60
 SECONDS_IN_MINUTE = 60
 
+T = TypeVar('T')
+U = TypeVar('U')
+DICT_LIST_TYPE = TypeVar('DICT_LIST_TYPE', Dict[Any, Any], List[Any])
```

Comment:
Why cannot you use `Union`?

Reply:
Yes I think DICT_LIST_TYPE can be replaced with union instead. Thanks!

Reply:
Done